### DISCUSSION TITLE
Can I ignore some folders?

### DISCUSSION THREAD

**Discussion by Smartyifan** [remotely-save/remotely-save#164] [Ideas]
I'd rather to sync a part of my vault than sync all of them, for example, I don't want to sync zotero folder through remotely-save, because it will exchange large amount of files and my webdav will be out of connect. It's perfect if I can ignore something through a ignore file like the .gitignore in git repo. Is this feature on the plan?

**Comment by fyears** [comment]
implemented in the latest version
